install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -32,6 +32,22 @@ Many storage types are available for use as persistent volumes in
 administrator, some types of storage are created dynamically using the
 built-in provider and plug-in APIs.
 
+[NOTE]
+====
+To enable dynamic provisioning, declare the
+`openshift_master_dynamic_provisioning_enabled` variable (within the `[OSEv3:vars]` section) in the Ansible
+inventory file and set its value to `True`.
```

Comment:
I'd say "To enable dynamic provisioning, add the `openshift_master_dynamic_provisioning_enabled` variable to the `[OSEv3:vars]` section of the Ansible inventory file and set its value to `True`."

IBM Style doesn't like parentheses.

Reply:
Thanks @kalexand-rh I'll fix this.